docs/tutorials/user/credit-class-project-batch-management.md

```diff
@@ -0,0 +1,468 @@
+
+For more information about the `data` module, see [Data Concepts](../../modules/data/01_concepts.md).
+
+### Define Resolver
+
+The following command will create a data resolver with a url of `[url]`. This is the URL at which you are hosting the data. You are claiming that given an IRI (e.g. `[url] + [iri]`), an application can fetch the data in complete or partial form depending on how you manage privacy.
+
+To create a data resolver, run the following command:
+
+```sh
+regen tx data define-resolver [url]
+```
+
+For more information about the command, add `--help` or check out [the docs](../../commands/regen_tx_data_define-resolver.md).
+
+Now that you created a data resolver, you can query it by id:
+
+```sh
+regen q data resolver [id]
+```
+
+### Content Hash
+
+Before we can anchor and register the data we created in the previous section, we must convert our IRIs into JSON objects representing content hashes on chain.
+
+To convert an IRI to a content hash, run the following command:
+
+```sh
+regen q data convert-iri-to-hash [iri]
+```
+
+For more information about the command, add `--help` or check out [the docs](../../commands/regen_query_data_convert-iri-to-hash.md).
+
+### Register Resolver
+
+The data for each credit class, project, and batch can now be registered to the resolver. The account that created the resolver is the only account that can register data to the resolver.
+
+With the next command, you can register all your data at once. You can register credit class, project, and batch data using the content hashes we converted from IRIs in the last step.
+
+To register data to a resolver, run the following command:
+
+```sh
+regen tx data register-resolver [resolver-id] [content-hashes-json]
+```
+For more information about the command, add `--help` or check out [the docs](../../commands/regen_tx_data_register-resolver.md).
+
+Now that you registered data to the resolver, you can look up resolvers by IRI:
+
+```sh
+regen q data resolvers-by-iri [iri]
+```
+
+If the data is in fact hosted at the location the data resolver is pointing to, the data can be fetched by Regen Network Development applications.
+
+## Credit Class
+
+Now that we have data for our class, project, and batch, we have IRIs for each, and we may or may not be hosting the data, we are ready to create a credit class.
+
+A credit class represents a collection of projects and issuers whereby the projects within a credit class are following the same standards and the issuers are issuing credits on their behalf.
+
+For more information about credit classes, see [Ecocredit Concepts](../../modules/ecocredit/01_concepts.md).
+
+### Allowlist and Fee
+
+
+Before we create a credit class, we need to check the credit class creator allowlist and that we have enough tokens for the credit class creation fee.
+
+To check whether the credit class creator allowlist is enabled, run the following command:
```

Comment:
```suggestion
The credit class creator allowlist gates credit class creation such that only a fixed list of addresses are permitted to create new credit classes. If the allowlist is disabled, then any user is able to create a new credit class on the network. To check whether the credit class creator allowlist is enabled, run the following command:
```

Reply:
This is additional context provided in the linked documentation. I'm hoping to reduce duplication. Maybe a link for more information would be better here.

Reply:
Added link https://github.com/regen-network/regen-ledger/pull/1965/commits/c5863d7f0fb55eab13f5cc7bfa0a86da841d5aa7